### DISCUSSION TITLE
Need to Metamask connect in my flutter app

### DISCUSSION THREAD

**Discussion by KamleshGESP** [WalletConnect/walletconnect-monorepo#3028] [Developer Support] [answered]
I have already connected metamask wallet to my flutter app, but currently stopped it. 
Reason : https://bridge.walletconnect.org not working

Please help me.

**Comment by crypblizz8** [comment, marked as answer]
Hey there, the bridge is no longer live due to the V1 Shutdown which you can read [here](https://medium.com/walletconnect/walletconnect-v1-0-has-now-been-shut-down-ff9baec96690).

We recommend you look into our [V2 flutter package. ](https://docs.walletconnect.com/2.0/flutter/installation)
